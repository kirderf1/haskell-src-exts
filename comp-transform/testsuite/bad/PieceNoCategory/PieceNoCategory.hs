{-# LANGUAGE ComposableTypes #-}

data piece Expr ==> Value = Const Int
