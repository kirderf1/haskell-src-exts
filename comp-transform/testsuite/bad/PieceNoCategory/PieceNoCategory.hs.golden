buildSigPiece: category UnQual () (Ident () "Expr") not declared.
